app/lib/models/patient_list.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:manv_api/api.dart';
+
+class PatientListEntry {
+  final String name;
+  final int id;
+
+  PatientListEntry({required this.name, required this.id});
+}
+
+typedef PatientList = List<PatientListEntry>;
+
+extension PatientListExtension on PatientList {
+  static PatientList fromApi(RunPatientAllIdsGet200Response? dto) {
+    if (dto == null) {
+      return [];
+    }
+
+    List<int> patientIds = dto.patientIds;
+    List<String> patientNames = dto.patientNames;
+
+    assert (patientIds.length == patientNames.length);
+
+    return List.generate(
+        patientIds.length,
+        (index) =>
+            PatientListEntry(name: patientNames[index], id: patientIds[index]));
```

Comment:
In retrospect i think this might be a good change, if you even have it like this in the model. (See my comment on the server code)

Reply:
see comment at server file